Dockerfile

```diff
@@ -1,27 +1,32 @@
 FROM rust:1.44.1-slim-buster as builder
 
-RUN apt-get update \
-  && apt-get install -y --no-install-recommends clang=1:7.* cmake=3.* \
-  libsnappy-dev=1.* curl \
-  && apt-get clean \
-  && rm -rf /var/lib/apt/lists/*
+WORKDIR /build
 
-RUN adduser --disabled-login --system --shell /bin/false --uid 1000 user
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends clang=1:7.* cmake=3.* \
+    libsnappy-dev=1.* \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
 
-WORKDIR /home/user
 COPY . .
-RUN chown -R user .
 
 RUN cargo install --locked --path .
 
 # Create runtime image
 FROM debian:buster-slim
 
-RUN adduser --disabled-login --system --shell /bin/false --uid 1000 user
+WORKDIR /app
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends curl \
```

Comment:
Thanks for the hint. I just checked and `wget` is not installed, because the base image is `debian:buster-slim`. One goal is to keep image sizes as small as possible. It's also the reason I split the build and runtime part in the Dockerfile.

Reply:
Sure, that makes sense. Keep `curl` then.